atmos.yaml

```diff
@@ -76,6 +76,16 @@ stacks:
     - "**/_defaults.yaml"
   # Can also be set using 'ATMOS_STACKS_NAME_PATTERN' ENV var
   name_pattern: "{tenant}-{environment}-{stage}"
+  list:
+    columns:
+      - name: Stack
+        value: '{{ .atmos_stack }}'
+      - name: Tenant
+        value: '{{ index .vars "tenant" }}'
+      - name: Environment
+        value: '{{ index .vars "environment" }}'
+      - name: File
+        value: '{{ .atmos_stack_file }}'
```

Comment:
We want our `atmos.yaml` to work for everyone by default.

Move this example to `examples/quick-start-advanced` and add it to the documentation as well, as an example of using it with the [Cloud Posse Reference Architecture](https://docs.cloudposse.com)